templates/security/disa-stig.html

```diff
@@ -1,190 +1,295 @@
 
 {% block meta_copydoc %}
-  https://docs.google.com/document/d/1zJwZzc-cERj9YKNXFtmrXwynJptyT7-D3qCNzBctRfo/edit#
+  https://docs.google.com/document/d/1zJwZzc-cERj9YKNXFtmrXwynJptyT7-D3qCNzBctRfo/edit?tab=t.0
 {% endblock meta_copydoc %}
 
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
 {% block content %}
 
-  <section class="p-strip--suru-topped">
-    <div class="row u-equal-height u-vertically-center">
-      <div class="col-8">
-        <h1>DISA-STIG on Ubuntu</h1>
-        <h2 class="p-heading--4">Comply with the DISA Security Technical Implementation Guide</h2>
-        <p>
-          Security Technical Implementation Guides (STIG) are developed by the Defense Information System Agency (DISA) for the U.S. Department of Defense (DoD). <a href="/cloud/public-cloud">Ubuntu Pro on public cloud</a> and <a href="/pro">Ubuntu Pro (Infra)</a> have the necessary <a href="/security/fips">certifications</a> and controls to comply with DISA-STIG guidelines on Linux.
-        </p>
-        <a class="p-button--positive js-invoke-modal"
-           href="/security/contact-us">Contact us</a>
-        <a class="p-button" href="/pro">Get Ubuntu Pro (Infra)</a>
-      </div>
-      <div class="col-4 u-hide--medium u-hide--small u-align--center">
+  {% call(slot) vf_hero(
+    title_text='DISA-STIG on Ubuntu',
+    subtitle_text='Comply with the DISA Security <br />Technical Implementation Guide',
+    layout='25/75'
+    ) -%}
+    {%- if slot == 'signpost_image' -%}
+      <div style="width:284px;">
         {{ image (
-        url="https://assets.ubuntu.com/v1/ef01809f-DISA-logo-transparent.png",
-        alt="DISA-STIG logo",
-        width="720",
-        height="264",
+        url="https://assets.ubuntu.com/v1/92754de5-disa.png",
+        alt="",
+        width="852",
+        height="204",
         hi_def=True,
         loading="auto"
         ) | safe
         }}
       </div>
-    </div>
-  </section>
-
-  <div class="u-fixed-width">
-    <hr class="p-rule" />
-  </div>
+    {%- endif -%}
+    {%- if slot == 'description' -%}
+      <p>
+        Security Technical Implementation Guides (STIG) are developed by the Defense Information System Agency (DISA) for the U.S. Department of Defense (DoD). <a href="/cloud/public-cloud">Ubuntu Pro on public cloud</a> and <a href="/advantage">Ubuntu Pro (Infra)</a> have the necessary <a href="/security/fips">certifications</a> and controls to comply with DISA-STIG guidelines on Linux.
+      </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="/security/contact-us"
+         class="p-button--positive js-invoke-modal">Contact us</a>
+      <a href="/advantage">Get Ubuntu Pro (Infra)&nbsp;&rsaquo;</a>
+    {%- endif -%}
+    {%- if slot == 'image' -%}
+      <div class="p-image-container--cinematic is-cover">
+        {{ image(url="https://assets.ubuntu.com/v1/f759a607-hero.png",
```

Comment:
This is a very large PNG. Could we try with fmt=jpg to reduce the size without losing to much data?

Reply:
AKAIK the [image template](https://github.com/canonical/canonicalwebteam.image-template/tree/main) is not able to take format as a parameter, I have made an [issue here to add this functionality](https://warthogs.atlassian.net/browse/WD-18977).

In the meantime, could we try a JPEG version of some of the bigger images on this page? @mattea-turic